### ISSUE TITLE
Semantic KITTI training configuration

### ISSUE THREAD

**Issue by billcai** [tsunghan-mama/ReDAL#7]
Hi authors, thanks for the interesting work. I am having trouble running the ReDAL benchmark for Semantic KITTI. I've processed the data according to the steps with no errors, but I'm hitting the following error when training:
```
/pytorch/aten/src/THCUNN/ClassNLLCriterion.cu:108: cunn_ClassNLLCriterion_updateOutput_kernel: block: [0,0,0], thread: [31,0,0] Assertion `t >= 0 && t < n_classes` failed.
```
I ran training with:
```
CUDA_VISIBLE_DEVICES=0 python3 train_region_active.py -n semantic_kitti -d <dataset location> --active_method ReDAL --ReDAL_config_path active_selection/ReDAL_SemanticKITTI.config --num_classes 19
```
I tried with num_classes as 20 as well and got the same error.

With some debugging, it appears that the max of `targets = batch['targets'].F.long().cuda(non_blocking=True)` = 255 at line 86 of `base_agent.py` which is more than the 19 classes of Semantic KITTI. I see the same range of values [0,255] for `labels` after adding print statements in `region_dataset.py`. Not sure if this is the intended range of `labels` but the mismatch in num classes and range of `labels` should not work when directly fed into `torch.nn.CrossEntropyLoss(ignore_index=self.args.ignore_idx)` in line 44 of `base_agent.py`.

**Comment by billcai** [comment]
I realized `--ignore_idx 255` is necessary for Semantic KiTTI. With this option, training works.

**Comment by Tony-WJR** [comment]
Hi, you have successfully reproduced ReDAL right?
I have a question, I noticed in the file ReDAL.py there are code like this:
outputs = model(inputs)
preds = outputs['final']
feats = outputs['pt_feat']
but outputs is the output of input passing network, and it is tensor type which do not have ['final'] or ['pt_feat'], do you know how to fix this? Thank you!

**Comment by Tony-WJR** [comment]
Hi billcai:
Have you successfully reproduced ReDAL? I found that when it comes to iteration 2, the training becomes extremely slow for 1 epoch (close to 1hour, which should not be like that), but in iteration 1 it only takes several minutes for 1 epoch. Have you met similar situations maybe?
